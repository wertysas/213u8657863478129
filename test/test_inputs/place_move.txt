PLACE 2,1,EAST
MOVE
REPORT