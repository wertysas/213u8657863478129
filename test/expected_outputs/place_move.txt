3,1,EAST